ansible/roles/jenkins-worker/vars/main.yml

```diff
@@ -5,7 +5,7 @@
 #
 
 init: {
-  aix: 'aix71',
+  aix: ['aix61', 'aix71', 'aix72'],
```

Comment:
If 6.1 is to remain in `/setup`, doesn't it just add confusion to add it here?

Or, if this can be used for 6.1, delete `/setup/aix61`?

Reply:
I removed 6.1. Should I remove 7.1? We don't have any 7.1 machines, so I doubt it would work (or has even been tested). Mind you, it might work once 7.2 is working.

Reply:
If it's reasonable to expect that 7.1 works, possibly with small modifications, I'm ok with this line. Everything in Ansible has to be taken with a grain of salt, we don't have the resources to ensure everything works all the time and this is not our final product in any case.

However if it's more likely that we'll never need this again in the future it's better to remove it, but no point in blocking this PR. So, it's your call!